ci/plugins/mzcompose/hooks/post-command

```diff
@@ -62,7 +62,8 @@ fi
 
 ci_unimportant_heading "Upload log artifacts"
 
-run logs --no-color > services.log
+# services.log might already exist and contain logs from before composition was downed
```

Comment:
This change is not needed I think and I am afraid I do not understand the comment.

You are deleting services log in the `command`, so `services.log` should not exist at this point. Therefore ,a single `>` should be enough at this stage?

Reply:
Oh sorry nevermind, you are manipulating services.log in both the bash part and the python part.